mergedSchema.js

```diff
@@ -24,8 +24,8 @@ function createConvectionLink() {
     if (tokenLoader) {
       return tokenLoader().then(({ token }) => {
         return {
-          ...context.headers,
           headers: {
+            ...context.headers,
```

Comment:
Maybe we don’t even really want to pass on (all) headers that a client specifies? I can imagine that we do want e.g. the request ID that @sweir27 had been working on.

Reply:
Yeah, we'd want the request ID - but that's probably it (at least in convection I've not seen any header handling outside of auth)

Reply:
Note: with #818 there's more headers we want to pass through